cmd/bee/cmd/db.go

```diff
@@ -492,109 +493,74 @@ func dbImportPinningCmd(cmd *cobra.Command) {
 	cmd.AddCommand(c)
 }
 
-//func dbNukeCmd(cmd *cobra.Command) {
-//	c := &cobra.Command{
-//		Use:   "nuke",
-//		Short: "Nuke the DB and the relevant statestore entries so that bee resyncs all data next time it boots up.",
-//		RunE: func(cmd *cobra.Command, args []string) (err error) {
-//			v, err := cmd.Flags().GetString(optionNameVerbosity)
-//			if err != nil {
-//				return fmt.Errorf("get verbosity: %w", err)
-//			}
-//			v = strings.ToLower(v)
-//			logger, err := newLogger(cmd, v)
-//			if err != nil {
-//				return fmt.Errorf("new logger: %w", err)
-//			}
-//			d, err := cmd.Flags().GetDuration(optionNameSleepAfter)
-//			if err != nil {
-//				logger.Error(err, "getting sleep value failed")
-//			}
-//
-//			defer func() { time.Sleep(d) }()
-//
-//			dataDir, err := cmd.Flags().GetString(optionNameDataDir)
-//			if err != nil {
-//				return fmt.Errorf("get data-dir: %w", err)
-//			}
-//			if dataDir == "" {
-//				return errors.New("no data-dir provided")
-//			}
-//
-//			logger.Warning("starting to nuke the DB with data-dir", "path", dataDir)
-//			logger.Warning("this process will erase all persisted chunks in your local storage")
-//			logger.Warning("it will NOT discriminate any pinned content, in case you were wondering")
-//			logger.Warning("you have another 10 seconds to change your mind and kill this process with CTRL-C...")
-//			time.Sleep(10 * time.Second)
-//			logger.Warning("proceeding with database nuke...")
-//
-//			localstorePath := filepath.Join(dataDir, "localstore")
-//			err = removeContent(localstorePath)
-//			if err != nil {
-//				return fmt.Errorf("localstore delete: %w", err)
-//			}
-//
-//			statestorePath := filepath.Join(dataDir, "statestore")
```

Comment:
This part also needs to be added. We will need to do few more things,
1. Check if forgetOverlay is set, in this case delete statestore completely
2. If not and forgetStamps is specified, we need to call Nuke on statestore. This function will need to be added to the storeadapter package in statestore package.
3. Also, we have 2 more directories in the new data-dir. If forgetStamps/forgetOverlay is specified, we should delete both these. You can check what directories are created by running a node.
4. In all cases, the `kademlia-metrics` can be deleted.